rhine/rhine.cabal

```
@@ -129,16 +129,15 @@ library
 
   -- Other library packages from which modules are imported.
   build-depends:
-                     , dunai        ^>= 0.9
+                     , dunai        >= 0.11
                      , transformers >= 0.5
                      , time         >= 1.8
                      , free         >= 5.1
                      , containers   >= 0.5
                      , deepseq      >= 1.4
                      , random       >= 1.1
                      , MonadRandom  >= 0.5
-                     -- Remove version pin when https://github.com/ivanperez-keera/dunai/issues/298 is resolved:
-                     , simple-affine-space == 0.1.1
+                     , simple-affine-space >= 0.2
```

```suggestion
                     , simple-affine-space ^>= 0.2
```
Same reasoning as for dunai